deployment/helm/resource-management-policies/balloons/templates/configmap_enable_nri.yaml

```
@@ -0,0 +1,31 @@
+{{- if .Values.nri.enable }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: patch-containerd-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "balloons-plugin.labels" . | nindent 4 }}
+data:
+  patch_nri.sh: |
+   #!/bin/bash
+
```

I think we should bail out on any error
```suggestion
set -e
```